modules/articles/tests/server/admin.article.server.routes.tests.js

```diff
@@ -23,7 +23,7 @@ var app,
 describe('Article Admin CRUD tests', function () {
   before(function (done) {
     // Get application
-    app = express.init(mongoose);
+    app = express.init(mongoose.connection.db);
```

Comment:
Yup! 

I guess https://github.com/lirantal/Riess.js might have quite a lot of it thought already.